pkg/repository/provider/unified_repo.go

```diff
@@ -47,18 +48,30 @@ var getGCPCredentials = repoconfig.GetGCPCredentials
 var getS3BucketRegion = repoconfig.GetAWSBucketRegion
 var getAzureStorageDomain = repoconfig.GetAzureStorageDomain
 
+type localFuncTable struct {
```

Comment:
Is the struct used for replacing the functions easily in unit tests? But seems these functions can receive mock input easily

Reply:
Yes, it is for mock the local functions for UT.
As the discussion, we will keep them as it for now, if we have better way in future, we come back to change it.

Reply:
Done